xla/stream_executor/dnn.h

```diff
@@ -1245,6 +1245,14 @@ class VersionInfo {
   int patch_;
 };
 
+class DnnGraph {
+ public:
+  virtual absl::StatusOr<bool> Prepare() = 0;
```

Comment:
`StatusOr<bool>` looks weird, I think plain `Status` should work here

Reply:
Ok, makes sense, can you add a bit of documentation because type signature alone is confusing

Reply:
Done.